### PR TITLE
include registerMethods examples in generated output.

### PR DESCRIPTION
**Pull request by jamestalmage**
It's a bit more verbose, but provides some helpful reminders about the API.

### CODE REVIEW

**Comment by jamestalmage** [comment]
@sindresorhus @jfmengels - I'd like your feedback if you think this more verbose example is good for the default output. It uses the `registerMethods` API.

**Comment by jfmengels** [comment]
Sorry, haven't taken the time to look at this repo. Will free some time tonight for it

File: generators/app/templates/codemod.js [hunk 1]
```diff
@@ -2,19 +2,33 @@
...
-		p.get('callee').replace(j.identifier('bar'));
 	});
 
-	return ast.toSource({
-		useTabs: true,
-		quote: 'single'
-	});
+	// Adding a method to all CallExpressions
+	j.registerMethods({
+		renameCallee: function (name) {
```
Comment (jfmengels) [inline generators/app/templates/codemod.js]: Can't you just do one call to `j.registerMethod`?

**Comment by jfmengels** [comment]
I'm not that familiar with `jscodeshift`'s API. I know the simple stuff mostly (and then again, I've forgotten most of it since my last real use of it), but I didn't know the `registerMethods` API.

I feel like this is a bit too complicated for beginners. I think that a simple "find and replace" is a good starter, without having to use `registerMethods` that look like they're needed for some heavy use cases. Then again, maybe this is a pretty great tool and I missed out on learning about it.

If you want to provide helper reminders, you could add links to the API docs of the things you think are useful or used often.

Another thing you can do to help the user is to give them an `astexplorer` link in some comment, so that they can play with it directly. I've prepared one here with the current default example https://astexplorer.net/#/rLeSwzcRGl (give a new one every time the example gets updated).

**Comment by jamestalmage** [comment]
> I feel like this is a bit too complicated for beginners.

Yeah, that was my main concern.

Maybe just put this in a separate template choice.

**Comment by sindresorhus** [comment]
> Another thing you can do to help the user is to give them an astexplorer link in some comment

👍 astexplorer is amazingly useful.

**Pull request closed without merging**